src/Vehicle/Vehicle.h

```diff
@@ -1031,6 +1034,9 @@ private slots:
     void _setMessageInterval            (int messageId, int rate);
     EventHandler& _eventHandler         (uint8_t compid);
 
+    int _obcHeartbeatCount = 0;
+    int _dlbHeartbeatCount = 0;
```

Comment:
```suggestion
    int _obcHeartbeatCount = 0;
    int _dlbHeartbeatCount = 0;
```
Comment the usecase of these variables